crates/red_knot/src/main.rs

```diff
@@ -28,30 +56,36 @@ pub fn main() -> anyhow::Result<()> {
     countme::enable(true);
     setup_tracing();
 
-    let arguments: Vec<_> = std::env::args().collect();
-
-    if arguments.len() < 2 {
-        eprintln!("Usage: red_knot <path>");
-        return Err(anyhow::anyhow!("Invalid arguments"));
+    let Args {
+        entry_point,
+        custom_typeshed_dir,
+        extra_search_path: extra_search_paths,
+        target_version,
+    } = Args::parse_from(std::env::args().collect::<Vec<_>>());
+
+    tracing::trace!("Checking file {entry_point}");
```

Comment:
This trace feels a bit extra verbose. I would remove it.

Reply:
Just this one, or the others as well?